--- lwp-2.6/configure.in~	2010-03-17 14:32:51.000000000 -0600
+++ lwp-2.6/configure.in	2010-07-17 10:48:14.000000000 -0600
@@ -14,7 +14,7 @@
 dnl   first to 0
 dnl - if any interfaces were added, increment third
 dnl - if any interfaces were removed, set third to 0
-CODA_LIBRARY_VERSION(9, 2, 0)
+CODA_LIBRARY_VERSION()
 
 CONFIG_DATE=`date +"%a, %d %b %Y %T %z"`
 AC_SUBST(CONFIG_DATE, "$CONFIG_DATE", [Date when configure was last run])
@@ -35,6 +35,11 @@
 dnl pkg-config stuff
 CODA_PKG_CONFIG
 
+CCASFLAGS="$ASFLAGS"
+CCAS="$AS"
+AC_SUBST(CCAS)
+AC_SUBST(CCASFLAGS)
+
 dnl Checks for libraries.
 AC_SUBST(LIBPTHREAD)
 AC_CHECK_LIB(pthread, pthread_create, [LIBPTHREAD="-lpthread"])
@@ -45,7 +50,7 @@
 AC_CHECK_TYPES(stack_t,,, [#include <signal.h>])
 
 dnl Checks for compiler characteristics
-CODA_CC_FEATURE_TEST(Wall)
+CODA_CC_FEATURE_TEST()
 
 dnl Checks for library functions.
 AC_CHECK_FUNCS(sigaltstack)
